### PR TITLE
Doppler experiment

### PR DESCRIPTION
**Pull request by SanliFaez**
@erikvansebille here is the code and some instructions for lecture 2

### CODE REVIEW

File: CourseDevelopment/Week2/Lecture2.md [hunk 1]
```diff
@@ -12,55 +11,39 @@ Course Development Week 2 for the Open Science Course for physics students at gr
...
 ## Tutorial: Week 2
-**Topics in tutorial:**
-- Between the two lectures, students will be given a complex coding task to complete in two teams, starting from forking an incomplete repository and then making the complete project work as a group! 
-- To make sure that every student practices the basic git commands, every student must be able to run the final program on their PC
-
-### keywords
-Unit testing (defensive programming) | Version control | Active and passive versioning (?) | Readme files and templates | Merging forked repositories
-
++ During the dutorial, students will be given a complex coding task to complete in multiple teams, starting from forking an incomplete repository and then making the complete project work as a group! 
++ To make sure that every student practices the basic git commands, every student is asked to to run the final program on their PC
```
Comment (erikvansebille) [inline CourseDevelopment/Week2/Lecture2.md:29]:
```suggestion
+ To make sure that every student practices the basic git commands, every student is asked to run the final program on their PC
```

File: CourseDevelopment/Week2/Lecture2.md [hunk 2]
```diff
@@ -12,55 +11,39 @@ Course Development Week 2 for the Open Science Course for physics students at gr
...
 
-The programming assignment will be based on analysing experimental data with the laptop sound card and deriving some commong physical properties. Elements of the code are provided to the students in a poorly-documented repository. In two separate teams, students should form a code production line that results in a working repository that works on every individual PC.
+The programming assignment will be based on analysing experimental data with the laptop sound card and doing an experiment with Doppler shift. Elements of the code are provided to the students in a [poorly-documented directory](../Week2/mycode). Students are divided into 4 teams of about 20+ and each team should make the code useful for analyzing a new measurement. The main goal is to create a well documented repository that works on every individual PC.
 
- 
 **Assignment preparation**
-  + Students have to discuss in a teams on distribution of task, timing and how they are working together
-  + >>> Further steps will be added when the assignment is ready <<<
+  + Students have to discuss in a teams on distribution of task, timing of their execution and how they are working and communication with each other.
```
Comment (erikvansebille) [inline CourseDevelopment/Week2/Lecture2.md:37]:
```suggestion
  + Students have to discuss in their teams on distribution of task, timing of their work, and how they will communicate during the work. 
```

File: CourseDevelopment/Week2/Lecture2.md [hunk 3]
```diff
@@ -12,55 +11,39 @@ Course Development Week 2 for the Open Science Course for physics students at gr
...
+The programming assignment will be based on analysing experimental data with the laptop sound card and doing an experiment with Doppler shift. Elements of the code are provided to the students in a [poorly-documented directory](../Week2/mycode). Students are divided into 4 teams of about 20+ and each team should make the code useful for analyzing a new measurement. The main goal is to create a well documented repository that works on every individual PC.
 
- 
 **Assignment preparation**
-  + Students have to discuss in a teams on distribution of task, timing and how they are working together
-  + >>> Further steps will be added when the assignment is ready <<<
+  + Students have to discuss in a teams on distribution of task, timing of their execution and how they are working and communication with each other. 
+  + Each team should present a code and code structure that can analyze newly recorded data in the shortest time and present a final result that has the highest accuracy.
+  + Each team will reserve one fresh laptop for the finale,
```
Comment (erikvansebille) [inline CourseDevelopment/Week2/Lecture2.md:39]:
```suggestion
  + Each team will reserve one fresh (i.e. no specific preparation) laptop for the finale,  
```

File: CourseDevelopment/Week2/mycode/doppler.py
Comment (erikvansebille) [inline CourseDevelopment/Week2/mycode/doppler.py:1]: This file is intentionally empty?

Comment (SanliFaez) [reply CourseDevelopment/Week2/mycode/doppler.py:1]: for now yes, but I may add a half-baked simulation of the signal as well. which may or maynot work :)

**Pull request merged**